sdk/identity/azure-identity/CHANGELOG.md

```diff
@@ -4,7 +4,6 @@
 
 ### Features Added
 - Added CAE support to service principal authentication.
-- Pass more detailed refresh policy for managed identity tokens to MSAL.
```

Comment:
do we need to mention that this was removed from previous betas?

Reply:
done